kyo-core/shared/src/test/scala/kyoTest/envsTest.scala

```diff
@@ -47,22 +47,8 @@ class envsTest extends KyoTest:
         succeed
     }
 
-    "intersection type env" - {
-        "explicit" in pendingUntilFixed {
-            val a = Envs.run(1)(Envs.get[Int & Double])
-            val b = Envs.run(1.2d)(a)
-            assert(b.pure == 1)
-            ()
-        }
-        "method inference" in pendingUntilFixed {
-            def test[T](v: T < (Envs[Int] & Envs[Double])) =
```

Comment:
Isn't this still relevant?

Reply:
Both scenarios fail because of the intersection tag and scenarios that test compilation failures are relatively fragile since they can break for different reasons as the codebase evolves. I don't see much value in keeping them. I'd say this is more of a `wontfix` for now but we can review if we explore supporting intersection and union types in tags.